README.md

```diff
@@ -93,6 +93,7 @@ Commands to run NEAR Lake, after `./target/release/near-lake`
 | `run`   	|                              	|                                                                  	| Runs the node                                                                                                                                                                                                                                                                                                                                                           	|
 |         	| `--bucket`                   	| Required                                                         	| AWS S3 Bucket name                                                                                                                                                                                                                                                                                                                                                      	|
 |         	| `--region`                   	| Required                                                         	| AWS S3 Bucket region                                                                                                                                                                                                                                                                                                                                                    	|
+|           | `--fallback-region`                  | Optional                                                          | AWS S3 Fallback region                                                                                                                                                                                                                                                                                                                                                 	|
```

Comment:
Could you please add info about what is the default value here?

Reply:
Yes, sure.